sample_aws_container/src/sample.py

```diff
@@ -0,0 +1,21 @@
+LIGHTRUN_KEY = 'LIGHTRUN_KEY'
+LIGHTRUN_SERVER = 'LIGHTRUN_SERVER_URL'
+
+def import_lightrun():
+    try:
+        import lightrun
+        lightrun.enable(com_lightrun_server=LIGHTRUN_SERVER, company_key=LIGHTRUN_KEY)
```

Comment:
I think you miss `lightrun_wait_for_init=true` that is required for lambdas, this will be released in 1.2.3, next week.
Probably need to change requirements too